tensorflow_probability/python/distributions/empirical_test.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2018 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Tests for the Empirical distribution."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import importlib
+
+# Dependency imports
+import numpy as np
+import tensorflow as tf
+
+from tensorflow_probability.python.distributions import empirical
+from tensorflow.python.framework import test_util
+
+
+def try_import(name):  # pylint: disable=invalid-name
```

Comment:
it doesn't look like this method ever gets used. if so, remove it.

Reply:
Fixed.